### PR TITLE
Feature/smart contract addresses in code

### PR DESCRIPTION
**Pull request by RafaelVidaurre**
**Please make sure the following boxes are checked before submitting your Pull Request:**

- [X] I linked the issues related to this PR in its description (if any).
- [X] If this PR adds new code that is not going to change in the near future, it includes unit tests to cover it.

This PR moves smart contract addresses to code:
- Hardcodes addresses per `deployment stage` + `network id`
- Most of this code is only meant to exist until we can transition to 2.0
- Modifies polymath.js' smart contracts to use hardcoded addresses unless explicitly passed an address
- Sets `REACT_APP_DEPLOYMENT_STAGE` to default to `local` 
- Adds tests for `constants` expected values per environment
- Sets up jest for `shared` repo

### CODE REVIEW

File: packages/polymath-js/src/contracts/TickerRegistry.js [hunk 1]
```diff
@@ -73,6 +73,7 @@ class TickerRegistry extends Contract {
 
   /**
    * TODO @bshevchenko: DEPRECATED since owner can be changed
+   * NOTE @RafaelVidaurre: Not really depracated, still being used
```
Comment (Grsmto) [inline packages/polymath-js/src/contracts/TickerRegistry.js]:
```suggestion
   * NOTE @RafaelVidaurre: Not really deprecated, still being used
```

File: packages/polymath-shared/babel.config.js [hunk 2]
```diff
@@ -0,0 +1,12 @@
+const base = require('../../config/babel.config.js');
+module.exports = {
+  babelrcRoots: ['../packages/*'],
+  presets: ['@babel/flow', '@babel/env', '@babel/react'],
```
Comment (Grsmto) [inline packages/polymath-shared/babel.config.js]: `@babel/react` this stuff might fail as it's not in the package.json. But except if you plan to have some React components in this package, you could remove this.

File: packages/polymath-js/src/contracts/TickerRegistry.js (hunk 1 above)
Comment (RafaelVidaurre) [reply packages/polymath-js/src/contracts/TickerRegistry.js]: Love the new feature <3

File: packages/polymath-shared/babel.config.js (hunk 2 above)
Comment (RafaelVidaurre) [reply packages/polymath-shared/babel.config.js]: True that, we don't need this for now 👍

**Pull request merged**